var/spack/repos/builtin/packages/py-ipywidgets/package.py

```diff
@@ -10,17 +10,22 @@ class PyIpywidgets(PythonPackage):
     """IPython widgets for the Jupyter Notebook"""
 
     homepage = "https://github.com/ipython/ipywidgets"
-    url      = "https://github.com/ipython/ipywidgets/archive/5.2.2.tar.gz"
+    pypi = "ipywidgets/ipywidgets-7.6.5.tar.gz"
 
-    version('7.5.1', sha256='e4253384886aabbaf10966916a2cf9ffa72551bd045d536fa2a379f14b50cec3')
-    version('7.4.2', sha256='f156165e8a855ed862fdf48e72700bdcd6956d089a2018c5b36d358255d45b2b')
-    version('5.2.2', sha256='d61ab8bb12b90981a3a6010429816d70eaa041e622043207bcb74239b664d4f3')
+    version('7.6.5', sha256='00974f7cb4d5f8d494c19810fedb9fa9b64bffd3cda7c2be23c133a1ad3c99c5')
+    version('7.5.1', sha256='e945f6e02854a74994c596d9db83444a1850c01648f1574adf144fbbabe05c97')
+    version('7.4.2', sha256='a3e224f430163f767047ab9a042fc55adbcab0c24bbe6cf9f306c4f89fdf0ba3')
+    version('5.2.2', sha256='baf6098f054dd5eacc2934b8ea3bef908b81ca8660d839f1f940255a72c660d2')
 
     depends_on('python@2.7:2.8,3.3:', type=('build', 'run'))
     depends_on('py-ipython@4:', type=('build', 'run'))
     depends_on('py-ipython@4:5', type=('build', 'run'), when='^python@:3.2')
+    depends_on('py-jupyterlab-widgets@1.0.0:', type=('build', 'run'),
```

Comment:
github said: "We couldn’t find any code matching 'py-jupyterlab-widgets' in spack/spack"

Reply:
Found it, sorry for overlooking it. It's merged in develop now. Can be merged now.